arboretum/kubernetes/README.md

```diff
@@ -17,7 +17,70 @@ to include the fetchers and checks from this library in your downstream project.
 
 ## Fetchers
 
-Fetchers coming soon...
+### Cluster Resource
+
+* Class: [ClusterResourceFetcher][fetch-cluster-resource]
+* Purpose: Write the resources of **stand-alone** Kubernetes clusters to the
+evidence locker.  **NOTE:** Do not use this fetcher for managed clusters.
+Instead use the [IBM Cloud cluster list fetcher][ibm-cloud-cluster-list-fetcher].
+* Behavior: Retrieve stand-alone Kubernetes cluster resource data for the provided
+list of clusters.  TTL is set to 1 day.
+* Configuration elements:
+  * `org.kubernetes.cluster_resources.clusters`
+    * Required
+    * List of dictionaries:
+      * `account name`: An arbitrary IBM Cloud account name as a string
```

Comment:
This should not be IBM Cloud specific, I think. Perhaps this field could be called `label` or `namespace`. Something general that could be applicable to any cloud provider. We only need to make sure it matches with what it is used in the credentials file.

Reply:
two stuffs:
1. the document contains "IBM Cloud", but it does not care about cloud service provider.  So I removed this description
2. changed the term `account` to `label` as you suggested, thanks. `namespace` or `name` would confuse kubernetes experts and thus I avoid using them.